processor/samplingprocessor/probabilisticsamplerprocessor/testdata/config.yaml

```diff
@@ -2,8 +2,21 @@ receivers:
   examplereceiver:
 
 processors:
+  # The probabilistic_sampler sets trace sampling by hashing the trace id of
+  # each span and making the sampling decision based on the hashed value. It
+  # also obeys the "sampling.priority" semantic convention as defined by
+  # OpenTracing. See
+  # https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table
```

Comment:
Should we mention that if "sampling.priority" is present it takes priority over hash-based decision?